ScaleformUI_Lua/src/LobbyMenu/MainView.lua

```diff
@@ -76,6 +76,9 @@ function MainView:Visible(visible)
         ScaleformUI.Scaleforms._pauseMenu:Visible(visible)
         if visible == true then
 			if not IsPauseMenuActive() then
+				self.focusLevel = 1
+				PlaySoundFrontend(-1, "Hit_In", "PLAYER_SWITCH_CUSTOM_SOUNDSET")
+				ActivateFrontendMenu(`FE_MENU_VERSION_EMPTY_NO_BACKGROUND`, true, -1)
```

Comment:
This is called twice?

Reply:
ActivateFrontendMenu is called once to activate pause menu and a second time to close it.. so yeah